sys/linux/sys.txt

```diff
@@ -385,8 +385,28 @@ iopl(level int8)
 ioperm(from intptr, num intptr, on intptr)
 ioprio_get$pid(which flags[ioprio_which_pid], who pid)
 ioprio_get$uid(which flags[ioprio_which_uid], who uid)
-ioprio_set$pid(which flags[ioprio_which_pid], who pid, prio intptr)
-ioprio_set$uid(which flags[ioprio_which_uid], who uid, prio intptr)
+ioprio_set$pid(which flags[ioprio_which_pid], who pid, ioprio flags[ioprio_priorities])
+ioprio_set$uid(which flags[ioprio_which_uid], who uid, ioprio flags[ioprio_priorities])
+
+# NEED: we can't express the construction of integers normally obtained using 
+# a combination of bitwise operations with the help of macros. For now, some
+# subset of valid values are enumerated as in ioprio_priorities.
```

Comment:
This can be expressed as a struct with flags fields though, right? The problem here is that this construction has to be passed a syscall argument directly. If so, please adjust the comment to clarify this.

Reply:
[fixed](https://github.com/google/syzkaller/pull/1919/files#diff-3377097d0218625c2406cd10cb77305aR391-R397)

Thanks for your review!